scripts/battlefields/Bearclaw_Pinnacle/when_hell_freezes_over.lua

```diff
@@ -0,0 +1,59 @@
+-----------------------------------
+-- When Hell Freezes Over
+-- Bearclaw Pinnacle ENM, Zephyr Fan
+-- !addkeyitem ZEPHYR_FAN
+-- !pos -757 -111 562 6
+-----------------------------------
+local ID = require("scripts/zones/Bearclaw_Pinnacle/IDs")
+require("scripts/globals/battlefield")
+require("scripts/globals/keyitems")
+require("scripts/globals/zone")
+-----------------------------------
+
+local content = Battlefield:new({
+    zoneId           = xi.zone.BEARCLAW_PINNACLE,
+    battlefieldId    = xi.battlefield.id.WHEN_HELL_FREEZES_OVER,
+    maxPlayers       = 18,
+    levelCap         = 75,
+    timeLimit        = utils.minutes(30),
+    index            = 2,
+    entryNpc         = "Wind_Pillar_3",
+    exitNpc          = "Wind_Pillar_Exit",
+    requiredKeyItems = { xi.ki.ZEPHYR_FAN, message = ID.text.ZEPHYR_RIPS },
+    grantXP          = 3000,
+})
+content.groups =
+{
+    {
+        mobs = { "Snow_Devil" },
+    },
+}
```

Comment:
`addEssentialMobs` will add a mob group with the listed mobs so you don't actually need to add a group with the mob in here unless you are going to add something else such as a mods/mobMods/callbacks/etc. In other words you can delete this.

Reply:
Thank makes sense.